help/de/Help_ch_Account-Actions.xml

```diff
@@ -514,92 +439,115 @@
-                Nachdem das Wertpapier angelegt wurde, wählen Sie den <guilabel>Typ</guilabel>
-                (normalerweise den Börsenplatz, an der das Wertpapier gehandelt wird) und den Namen
-                in der Dropdown-Liste <guilabel>Währung/Wertpapier</guilabel> und
-                <guibutton>Schließen</guibutton> den Dialog.
-              </para>
-            </listitem>
-          </itemizedlist>
-        </listitem>
+          </listitem>
+        </varlistentry>
 
-        <listitem>
-          <para><guilabel>Kleinste Stückelung:</guilabel> Die kleinste Teilung, die erfasst werden kann.
-          </para>
-        </listitem>
+        <varlistentry>
+          <term><guilabel>Wertpapier/Währung</guilabel></term>
 
-        <listitem>
-          <para><guilabel>Kontofarbe:</guilabel> Die Farbe, die der Registerkarte des Kontos zugewiesen werden soll.
-            Klicken Sie auf die Schaltfläche <guibutton>Farbe</guibutton>, um die Anzeige
-            <guilabel>Farbe auswählen</guilabel> zu öffnen. Wollen Sie die Farbeinstellung für
-            das Kontos zurückzusetzen betätigen Sie die Schaltfläche
-            <guibutton>Voreinstellung</guibutton>.
-          </para>
-        </listitem>
+          <listitem>
+            <note>
+              <para>Der <guilabel>Typ</guilabel> im Feld des <guilabel>Wertpapier auswählen</guilabel>-Dialogs ist
+                abhängig von der Auswahl im <guilabel>Kontoart</guilabel>-Feld in der
+                <guilabel>Neues Konto</guilabel> Anzeige.
+              </para>
+            </note>
 
-        <listitem>
-          <para><guilabel>Bemerkung:</guilabel> Dies ist ein Freitextfeld, das für zusätzliche Notizen über das
-            Konto verwendet werden kann.
-          </para>
-        </listitem>
-      </orderedlist>
+            <itemizedlist spacing="compact">
+              <listitem>
+                <para>Für die meisten Konten wird hier die Standardwährung, d.h. Ihre Landeswährung, z.B. <quote>EUR
+                  (Euro)</quote> angezeigt. Falls das Konto für eine Fremdwährung vorgesehen ist,
+                  benutzen Sie die <guibutton>Auswählen&hellip;</guibutton> Schaltfläche und
+                  wählen im folgenden Auswahldialog eine andere Währung.
+                </para>
+              </listitem>
+
+              <listitem>
+                <para>Für Konten, die <emphasis>Wertpapiere</emphasis> enthalten, wählen Sie zunächst
+                  <guilabel>Aktie</guilabel> oder <guilabel>Investmentfonds</guilabel> in dem Feld
+                  <guilabel>Kontoart</guilabel> aus, dann das <guilabel>Hauptkonto</guilabel> und im
+                  Anschluss benutzen Sie die Schaltfläche
+                  <guibutton>Auswählen&hellip;</guibutton>, um den <guilabel>Typ</guilabel>
+                  auszuwählen (in der Regel den Börsenplatz an dem das Wertpapier gehandelt wird)
```

Comment:
Ich halte den Börsenplatz immer noch für einen der dümmsten Fehler in der Doku, der anscheinend nicht totzukriegen ist.
Ich empfehle als erste Ebene Typ des Wertpapiers: Bond, Aktie, Fond, Option …
als optionale 2. Ebene die Nominalwährung.

Bond:EUR
Bond:CHF
Aktie:EUR
Aktie:CHF
…

Reply:
@fellen: auch hier fehlt mir die Fantasie was wie ich das besser beschreiben kann. Kannst du mir einen Vorschlag unterbreiten?

Reply:
Ähnlich der Kontonummer bietet der Typ, der intern Namensraum heißt, eine Möglichkeit, seine Investments sachlich zu sortieren.

Anaschließend kannst du meine obige Empfehlung in die 3. Person umformulieren.

Reply:
ok. Ich habe mich an einem anderen Formulierung versucht.